Templates/Ros2RoboticManipulationTemplate/README.md

```diff
@@ -4,7 +4,14 @@ This project is a template designed to help you quickly start simulating robotic
 
 This template focuses on robotic arm manipulation with ROS 2 and [MoveIt 2](https://moveit.picknik.ai/main/index.html).
 
+## Prerequisites 
 
+Make sure that your system meets [Requirements](https://development--o3deorg.netlify.app/docs/welcome-guide/requirements/).
+
+Install ROS2 dependencies for this template:
+```bash
+sudo apt install ros-humble-moveit ros-humble-moveit-resources ros-humble-depth-image-proc
```

Comment:
Is this template limited to ROS2 Humble or should we allow different versions as well?
```suggestion
sudo apt install ros-${ROS_DISTRO}-moveit ros-${ROS_DISTRO}-moveit-resources ros-${ROS_DISTRO}-depth-image-proc
```